src/portable_python/cpython.py

```diff
@@ -228,7 +228,7 @@ def _finalize(self):
 
                 self.run_python(cmd)
 
-            self.run_python("-mpip", "install", *runez.flattened(additional))
+            self.run_python("-mpip", "install", "--no-cache-dir", "--upgrade", *runez.flattened(additional))
```

Comment:
So sorry it took me so long to get around this, the tests are failing because we need to update `test_setup.py` line 14, because it is currently looking for a mention of `install my-additional-package` and the chatter has changed, so:

```
assert " -mpip install --no-cache-dir --upgrade my-additional-package" in cli.logged
```

Reply:
I'll merge it and fix the test accordingly